src/tabs/firmware_flasher.html

```diff
@@ -161,16 +161,16 @@
                 <div class="margin-bottom">
                     <div style="width: 49%; float: left;">
                         <strong i18n="firmwareFlasherBuildRadioProtocols"></strong>
-                        <div id="radioProtocolInfo">
-                            <select id="radioProtocols" name="radioProtocols" multiple="multiple" class="select2" style="width: 95%">
+                        <div id="radioProtocolInfo" class="build-options-wrapper">
```

Comment:
Can we define this class once?
Maybe add to `build_configuration` class

Reply:
can access all these elements via something like
`.build_configuration .spacer div div:first-child`

but way cleaner to just give it a descriptive class name